src/components/Videos.vue

```diff
@@ -42,6 +42,15 @@
 				@canplay="doneLoading"
 				@loadedmetadata="onLoadedMetadata">
 
+				{{ fetchTracks() }}
```

Comment:
Templates are not made to call methods, please use the `watch: active` method.

Reply:
I'm working on this atm. Moved it to onFetchedMetadata

Reply:
I take that back, I cannot push to your branch apparently. But as @skjnldsv is right. I fixed it like this:

- In `onLoadedMetadata`: add a call to `this.fetchTracks()`
- Remove the `{{ fetchTracks() }}`

Reply:
Thanks @skjnldsv 
Will be making the requested changes.

Reply:
@beardhatcode Yes, moving it to 'onLoadedMetadata' works pretty good !